SAKPaaS/src/app/components/occupancy-view/occupancy-view.component.ts

```diff
@@ -60,7 +60,7 @@ export class OccupancyViewComponent implements OnInit, OnDestroy {
   }
 
   getStringForOcc(low: string, medium: string, high: string, noData: string): string {
-    if (!this.occupancy.value || this.occupancy.value < 0) {
+    if (this.occupancy.value === null || this.occupancy.value === undefined || isNaN(this.occupancy.value) || this.occupancy.value < 0) {
```

Comment:
Es stört mich nicht sonderlich, aber warum `isNaN`? Wo soll denn da ein `NaN` herkommen? Und inwiefern ist das dann besser als `!this.occupancy.value`?

Reply:
Wir hatten letztens das Problem, dass das Backend ein `NaN` zurückgegeben hat und dann wurde immer höchste Auslastung angezeigt. Wollte das unterbinden, falls es nochmal auftreten sollte.

Und das Problem bei `!this.occupancy.value` ist, dass es auch auf `0` vergleicht. Mittlerweile kann die Occupancy auch `0` sein ('keine Auslastung' schickt ja jetzt auch `0` als Wert).
Deswegen leider die komplexere Abfrage ... @timgrohmann

Reply:
Okay dann lgtm.